src/App.css

```diff
@@ -0,0 +1,64 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
+
+body {
+  font-family: "Roboto", sans-serif;
+  background-color: #F5F5F5;
+}
+
+.container {
+  display: inline-flex;
+  padding: 19px 102px 20px 100px;
+  justify-content: center;
+  flex-shrink: 0;
+}
+
+.logo {
```

Comment:
Create a separe css file for the navbar please. Remove the CSS code for the navbar from app.css